app/models/service.rb

```diff
@@ -0,0 +1,7 @@
+class Service < ApplicationRecord
+  belongs_to :major
+  has_many :review
```

Comment:
Đã has_many thì đặt tên số nhiều e nhé
```suggestion
  has_many :reviews
```

Tương tự fix hết